api/src/main/java/org/geysermc/geyser/api/pack/ResourcePack.java

```diff
@@ -26,12 +26,15 @@
 package org.geysermc.geyser.api.pack;
 
 import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
  * Represents a resource pack sent to Bedrock clients
  * <p>
  * This representation of a resource pack only contains what
  * Geyser requires to send it to the client.
```

Comment:
```suggestion
 * Represents a resource pack sent to Bedrock clients.
 * <p>
 * This representation of a resource pack tends to be minimal.
```
Feel free to word it differently